packages/components/addon/components/hds/app-footer/status-link.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export const STATUSES = {
+  OPERATIONAL: {
+    text: 'Operational',
+    iconName: 'check-circle-fill',
+    iconColor: 'var(--token-color-foreground-success)',
+  },
+  DEGRADED: {
+    text: 'Degraded',
+    iconName: 'alert-triangle',
+    iconColor: 'var(--token-color-foreground-warning)',
+  },
+  OUTAGE: {
+    text: 'Outage',
+    iconName: 'x-square-fill',
+    iconColor: 'var(--token-color-foreground-critical)',
+  },
+  MAINTENANCE: {
+    text: 'Maintenance',
+    iconName: 'info-fill',
+    iconColor: 'var(--token-color-foreground-highlight)',
+  },
+};
+
+export default class HdsAppFooterStatusLinkComponent extends Component {
+  get status() {
+    return this.args.status.toUpperCase();
```

Comment:
what happens if as consumer I pass the status `foo`?

Reply:
I updated this so that they will get an error if they pass a non-standard status. We can refactor further if this isn't desired behavior.

Reply:
Resolving as I handled this use case.